src/deadline/nuke_submitter/default_nuke_job_template.yaml

```diff
@@ -25,6 +25,11 @@ parameterDefinitions:
     label: Gizmo Directory
   description: The directory containing Nuke Gizmo files used by this job.
   default: './gizmos'
+- name: OCIOConfigPath
+  type: PATH
+  objectType: FILE
+  dataFlow: IN
+  description: The directory containing OCIO config files used by this job.
```

Comment:
This looks to be just a single FILE not a directory like the description says. One or the other probably needs to be updated.

Reply:
Thanks for catching, I will update this to something like, 
`description: The OCIO config file used by this job`